packages/frontend/src/components/badge/NotSyncedBadge.tsx

```diff
@@ -5,14 +5,23 @@ import { Tooltip, TooltipContent, TooltipTrigger } from '../tooltip/Tooltip'
 import { Badge } from './Badge'
 
 interface NotSyncedBadgeProps {
-  displaySyncedUntil: string
+  displaySyncedUntil?: string
   className?: string
 }
 
 export function NotSyncedBadge({
   displaySyncedUntil: syncedUntil,
   className,
 }: NotSyncedBadgeProps) {
+  if (!syncedUntil)
+    return (
+      <div className={cn('inline', className)}>
+        <Badge size="extraSmall" padding="regular" type="gray">
+          NOT SYNCED
+        </Badge>
```

Comment:
maybe lets extract it to separate fn on the bottom of file to use it in tooltip and here